components/server/src/api/organization-service-api.ts

```diff
@@ -49,18 +51,20 @@ export class OrganizationServiceAPI implements ServiceImpl<typeof OrganizationSe
         private readonly apiConverter: PublicAPIConverter,
     ) {}
 
-    async createOrganization(
-        req: CreateOrganizationRequest,
-        context: HandlerContext,
-    ): Promise<CreateOrganizationResponse> {
-        const org = await this.orgService.createOrganization(context.user.id, req.name);
+    async createOrganization(req: CreateOrganizationRequest, _: HandlerContext): Promise<CreateOrganizationResponse> {
+        // TODO(gpl) This mimicks the current behavior of adding the subjectId as owner
+        const ownerId = ctx().subjectId?.userId();
```

Comment:
same here. I'd prefer a global `currentUserId()` that encapsulates the access to the ctx.

Reply:
I see you have introduced that and marked it as deprecated. I think the function declaration is actually good (incl. the excpetion it throws) but we should not require the first argument on all service methods in the long run (i.e. deprecate that somehow)